File: 프로그래머스/lv0/120824. 짝수 홀수 개수/짝수 홀수 개수.py
def solution(num_list):
    answer = []
    a_list = []
    b_list = []
    for a in num_list:
        if a % 2 == 0:
            a_list.append(a)
        else:
            b_list.append(a)
    a_num = len(a_list)
    b_num = len(b_list)
    c_list =[]
    c_list.append(a_num)
    c_list.append(b_num)
    answer.extend(c_list)
        
    return answer

File: 프로그래머스/lv0/120824. 짝수 홀수 개수/짝수 홀수 개수.js
function solution(num_list) {
    var answer = [0, 0];
    let lenNum = num_list.length
    for(i = 0; i <= lenNum - 1; i++){
        if(num_list[i] % 2 == 0){
            answer[0]++
        }
        else{
            answer[1]++
        }
    }
    return answer;
}